packages/commonwealth/server/migrations/20230911194726-cosmos-chain-id.js

```diff
@@ -0,0 +1,80 @@
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'csdkbetaci' WHERE name = 'cosmos sdk v0.46.11';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'crescent' WHERE name = 'Crescent Network';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'evmos' WHERE name = 'Evmos';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'evmosdevci' WHERE name = 'evmos dev ci';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'evmosdev' WHERE name = 'evmos sandbox';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'haqq' WHERE name = 'Haqq Network';
+
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'injective' WHERE name = 'Injective (Mainnet)';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'jackal' WHERE name = 'Jackal Protocol';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'juno' WHERE name = 'Juno';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'kava' WHERE name = 'Kava';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'kichain' WHERE name = 'KiChain';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'kyve' WHERE name = 'KYVE Network';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'odin' WHERE name = 'ODIN Protocol';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'oraichain' WHERE name = 'Oraichain';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'osmosis' WHERE name = 'Osmosis';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'panacea' WHERE name = 'Panacea';
+
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'persistence' WHERE name = 'Persistence';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'planq' WHERE name = 'Planq';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'quasar' WHERE name = 'Quasar Finance';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'quicksilver' WHERE name = 'Quicksilver Protocol';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'qwoyn' WHERE name = 'Qwoyn Network';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'regen' WHERE name = 'Regen Network';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'secretnetwork' WHERE name = 'Secret Network';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'sei' WHERE name = 'sei';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'sentinel' WHERE name = 'Sentinel';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'shentu' WHERE name = 'Shentu Chain';
+
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'sifchain' WHERE name = 'Sifchain';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'sommelier' WHERE name = 'Stafihub';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'stargaze' WHERE name = 'Stargaze';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'terpnetwork' WHERE name = 'Terp Network ';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'terra' WHERE name = 'Terra Classic ';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'umee' WHERE name = 'Umee';
+        UPDATE "ChainNodes" SET cosmos_chain_id = 'vidulum' WHERE name = 'Vidulum (Mainnet)';
+        `,
```

Comment:
We can add "Gravity Chain": https://cosmos.directory/gravitybridge

Reply:
You're right. I thought the Althea entry was gravity bridge. They are the team that built it I believe. Fixed.